dracut/30ignition/ignition-files.service

```diff
@@ -1,26 +1,28 @@
 [Unit]
 Description=Ignition (files)
 DefaultDependencies=false
-Before=initrd-parse-etc.service
-
-Requires=initrd-root-fs.target
-After=initrd-root-fs.target
+# This isn't the /etc you're looking for.
+#Before=initrd-parse-etc.service
```

Comment:
that unit contains

```
ExecStartPre=-/usr/bin/systemctl daemon-reload
# we have to retrigger initrd-fs.target after daemon-reload
ExecStart=-/usr/bin/systemctl --no-block start initrd-fs.target
ExecStart=/usr/bin/systemctl --no-block start initrd-cleanup.service
```
why is ostree special such that `initrd-parse-etc.service` is not needed for ostree but is needed for non-ostree ?

Reply:
I think it's *probably* fine with it enabled as well, but it's not much use because at that point, `/sysroot/etc` is not yet pointing to the deployment root `/etc`. Looking at the documentation for this [here](https://www.freedesktop.org/software/systemd/man/bootup.html), it mentions reading `/etc/fstab` to find `/usr` mounts, which we don't support anyway.

Reply:
do you mind adding that comment in there?